pkgs/jnigen/test/jackson_core_test/generate.dart

```diff
@@ -38,7 +40,7 @@ Config getConfig({
     mavenDownloads: MavenDownloads(
       sourceDeps: deps,
       sourceDir: join(thirdPartyDir, 'java'),
-      jarDir: join(thirdPartyDir, 'jar'),
+      jarDir: File(join(thirdPartyDir, 'jar')).absolute.path,
```

Comment:
Why was this needed?

Reply:
It might be a problem only in testing but the resolution of the path was sometimes unpredictable and was getting confused using a wrong working directory.